documentation/release_notes.markdown

```diff
@@ -14,6 +14,108 @@ canonical: "/puppetdb/latest/release_notes.html"
 [metrics]: ./api/metrics/v1/changes-from-puppetdb-v3.html
 [pqltutorial]: ./api/query/tutorial-pql.html
 
+4.2.0
+-----
+
+PuppetDB 4.2.0 is a backward-compatible feature release that adds a
+new inventory endpoint, the ability to query structured data like
+facts and resource parameters using a dotted notation and the ability
+to trigger background/GC tasks manually via POST. This release
```

Comment:
I wondered if "structured data (like facts and resource parameters) ..." might read a little easier.  For some reason I kept reading it a little differently without that.